test/test_file.py

```diff
@@ -0,0 +1,36 @@
+from mathics.core.parser import parse, SingleLineFeeder
+from mathics.core.definitions import Definitions
+from mathics.core.evaluation import Evaluation
+import pytest
+
+
+definitions = Definitions(add_builtin=True)
+evaluation = Evaluation(definitions=definitions, catch_interrupt=False)
+
+
+def _evaluate(str_expression):
+    expr = parse(definitions, SingleLineFeeder(str_expression))
+    return expr.evaluate(evaluation)
+
+
+def test_get_and_put():
+    temp_directory = _evaluate("$TemporaryDirectory").to_python()
+    if len(temp_directory)<3:
+        return
+    temp_directory = temp_directory[1:-1]
+    temp_filename = temp_directory + "/testfile"
+    print(temp_filename)
+    result = _evaluate("40! >> " + temp_filename).to_python()
+    assert result is None
+
+    result = _evaluate("<< " + temp_filename)
+    assert result == _evaluate("40!")
+
+    result = _evaluate("DeleteFile[\"" + temp_filename + "\"]").to_python()
```

Comment:
This shoud probably be an f-string (`f"DeleteFile[\"{temp_filename}\"]"`)

Reply:
Not a big deal thought, we can merge the PR. Down the line I'd like to covert this (and every other instance of string interpolation done with C-style format strings or the `+` operator) to an f-string.

Reply:
Why? I don't get used to use fstrings. What is the advantages of use them?

Reply:
> Why? I don't get used to use fstrings. What is the advantages of use them?

Well, f-strings are the "official" way to do string interpolation in Python nowadays. They are more legible than concatenating strings, and they are more flexible than the printf-like syntax used in the `%` operator (which used to be the "official" way to interpolate strings in Python 2.x).

The example in question doesn't really illustrate this points well enghout, but image you're trying to interpolate 4 values into a string. You can do it the "concatenation route" as you discribed:

```py
"blabla" + x + "bleble" + y + "blibli" + z + "bloblo" + w + "blublu"
```
Or you can do it with f-strings:
```py
f"blabla{x}bleble{y}blibli{z}bloblo{w}blublu"
```

In my opinion, the f-string is a lot more readable than the concatenation thing (I can get a much clearer picture of what the interpolated string is going to look like). F-strings are also safer, for example, `"a" + x + "b"` raises a `TypeError` when `x` is not a string, while `f"a{x}b"` doesn't (`f"a{x}b"` is equivalent to `"a" + str(x) + "b"`). You can find more information on f-strings in [here](https://realpython.com/python-f-strings/).